lib/absinthe/plug/request.ex

```
@@ -50,12 +50,18 @@ defmodule Absinthe.Plug.Request do
     end
   end
 
+  defp parse_json_params(json) when is_list(json), do: json
+  
+  defp parse_json_params(json), do: Poison.decode!(json)
```

This will also need to return a nice error if it does not decode.